v63004/u_inref/gtm8909.exp

```diff
@@ -24,10 +24,13 @@ proc timeout_procedure { } {
 	exit -1
 }
 
+send -- "stty cols 130\r"
```

Comment:
Add a comment before this line as to WHY this is done (i.e. something like what we have in r120/u_inref/ctrlchandler.exp).

Reply:
An explanatory comment has been added